astropy/visualization/wcsaxes/coordinates_map.py

```diff
@@ -96,7 +96,10 @@ def __init__(self, axes, transform=None, coord_meta=None,
             # Set up aliases for coordinates
             if isinstance(name, tuple):
                 for nm in name:
-                    self._aliases[nm] = index
+                    # Do not replace an alias already in the map if we have
+                    # more than one alias for this axis.
+                    if nm.lower() not in self._aliases:
```

Comment:
remove lower? or check that both the lowercase and original are not in aliases

Reply:
line 102 below is the only place we add things to `self._aliases` and we do it with a lower?

Reply:
line 102 doesn't show a lower, only line 104 does?

Reply:
that's what I meant :facepalm:

Reply:
But my point is that on line 102 we also add things to aliases *without* a ``lower``? Shouldn't both lines 102 and 104 use lower?

Reply:
I just added the lower to all the things?